diff --git a/qt/aqt/__init__.py b/qt/aqt/__init__.py
index b6d24080b..749145c67 100644
--- a/qt/aqt/__init__.py
+++ b/qt/aqt/__init__.py
@@ -6,7 +6,7 @@ from __future__ import annotations
 import logging
 import sys
 
-import pip_system_certs.wrapt_requests
+# import pip_system_certs.wrapt_requests
 
 if sys.version_info[0] < 3 or sys.version_info[1] < 9:
     raise Exception("Anki requires Python 3.9+")
@@ -413,12 +413,6 @@ def setupGL(pm: aqt.profiles.ProfileManager) -> None:
     # RHI errors are emitted multiple times so make sure we only handle them once
     driver_failed = False
 
-    # work around pyqt loading wrong GL library
-    if is_lin and not sys.platform.startswith("freebsd"):
-        import ctypes
-
-        ctypes.CDLL("libGL.so.1", ctypes.RTLD_GLOBAL)
-
     # catch opengl errors
     def msgHandler(category: Any, ctx: Any, msg: Any) -> None:
         if category == QtMsgType.QtDebugMsg:
--
2.24.0
