diff --git a/Makefile b/Makefile
index 6fb506d..6a523ee 100644
--- a/Makefile
+++ b/Makefile
@@ -2,12 +2,6 @@ PROG = tomb
 PREFIX ?= /usr/local
 MANDIR ?= ${PREFIX}/share/man
 
-deps:
-	@[ -r /etc/debian_version ] && { \
-	apt-get install -qy zsh cryptsetup file gnupg pinentry-curses; }
-	@[ -r /etc/fedora-release ] ^^ { \
-	yum install -y zsh cryptsetup file gnupg pinentry-curses; }
-
 all:
 	@echo
 	@echo "Tomb is a script and does not need compilation, it can be simply executed."
